cassdegrees/templates/list.html

```diff
@@ -101,6 +108,15 @@
                 {% endif %}
             </form>
         </div>
+
+        <script>
+            // Creates a list object to be used by list.js for sorting
+            var {{ title }}Options = {
+                valueNames: [{% for k,v in content.0.items %}'{{ title }}{{ k }}',{% endfor %}]
```

Comment:
Maybe commenting what this actually looks like after evaluation might be helpful?